docs-kits/kits/Connector Kit/Adoption View/policies-in-catena.md

```diff
@@ -0,0 +1,253 @@
+
+This guidance is however also relevant for Enablement Service Providers building components enabling connectivity to the
+Dataspace (as specified in CX-0018). The authoritative resource for schemas is
+the [Catena-X ODRL Profile](https://github.com/catenax-eV/cx-odrl-profile). It is also available via the [Policy Hub](https://github.com/eclipse-tractusx/policy-hub/blob/main/docs/technical-documentation/requests/example-requests.md)
+that is operated centrally. The API is documented in this Repository and can be accessed with an access token to the
+Portal. It's a convenience feature for the negotiating parties to check if a given offer matches the policy constraints
+agreed by the Catena-X association - for instance by keeping a definite list of valid `rightOperands` to a particular
+`leftOperand` in a `Constraint`.
+
+As mentioned in the primer on policies, Providers and Consumers must have a common
+understanding of the meaning and consequences of `odrl:Offers` and, on a more granular level, their `odrl:Constraints`.
+That's why there is a set of predefined `odrl:Constraints` - all of which have to
+be [accepted explicitly](working-with-policies.md#consumer-side-odrloffer-in-a-contractrequestmessage) and
+some [checked against a Consumer's VP](working-with-policies.md#provider-side-checking-a-consumers-verifiable-presentation)
+additionally. They are formalized in the [Catena-X ODRL profile](https://github.com/catenax-eV/cx-odrl-profile)
+which extends the regular [ODRL vocabulary](https://www.w3.org/TR/odrl-vocab/). Since usage or contract policies are
+highly dependent on the use case,
+they are described by them in their associated KITs and only general elements are explained in the following.
+
+Here's a non-normative overview of these extensions:
+
+### Use Case Framework Constraints
+
+Use Case Framework Constraints are references to legally binding documents set up by the Catena-X association. They
+govern the _"who, with whom, what, where from and where to, why, how, and when"_ of Data Sharing in Catena-X Use-Cases
+([Source](https://catena-x.net/en/catena-x-introduce-implement/governance-framework-for-data-space-operations)).
+Use Case Frameworks are roughly structured along the lines of business scenarios under which a set of business partners
+want to exchange data.
+
+Each Participant commits to a set of Use Case Frameworks during Onboarding. They are granted a set of VCs as proof of
+that commitment. Consequently, Use Case Framework Constraints belong to the kind of `odrl:Constraint`s that have to be
+[checked against a VP](working-with-policies.md#provider-side-checking-a-consumers-verifiable-presentation). The
+complete set is listed in the most current version of standard
+[CX-0050 Framework Credential](https://catena-x.net/fileadmin/user_upload/Standard-Bibliothek/Update_September23/CX-0050-FrameworkAgreementCredential-v.1.0.0.pdf) (
```

Comment:
Please update to V 2.0.0 (for 24.05 in the opt out phase - link will be outdated in a few weeks - suggest, linking to the overall standard library)